docs/_sass/_fonts.scss

```diff
@@ -1,95 +1,46 @@
 @font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 100;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Light.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 200;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Thin.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
+  font-family: ExpensifyNeue-Regular;
   font-weight: 400;
   font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Regular.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Regular.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Regular.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 500;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: GTAmericaExp-Regular;
-  font-weight: 600;
-  font-style: normal;
-  src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2");
+  src: url('/fonts/ExpensifyNeue-Regular.woff') format('woff'), url('/fonts/ExpensifyNeue-Regular.woff2') format('woff2');
```

Comment:
Did you test this? The Jekyll site doesn't go through the same webpack bundler as the main site, so I don't see any reason why the fonts would be hosted at `https://help.expensify.com/fonts`

Reply:
Oh, yeah, that might need to be reverted. I'll give it a spin